src/decryptor.rs

```diff
@@ -0,0 +1,53 @@
+use sdkms::{
+    api_model::{Algorithm::Rsa, DecryptRequest, SobjectDescriptor},
+    SdkmsClient,
+};
+use sequoia_openpgp::{
+    crypto::{mpi, Decryptor, SessionKey},
+    packet::{
+        key::{PublicParts, UnspecifiedRole},
+        Key,
+    },
+    Result as SequoiaResult,
+};
```

Comment:
Minor remark: we usually don't group imports like this. I don't think we have the rule written down anywhere, but in other crates we only group imports at the deepest level. What I mean is, in this case:
```
use sdkms::api_model::Algorithm::Rsa;
use sdkms::api_model::{DecryptRequest, SobjectDescriptor};
use sdkms::SdkmsClient;
use sequoia_openpgp::crypto::{mpi, Decryptor, SessionKey};
use sequoia_openpgp::packet::key::{PublicParts, UnspecifiedRole};
use sequoia_openpgp::packet::Key;
use sequoia_openpgp::Result as SequoiaResult;
```

Reply:
Great, it is way more neat like this. Too bad that `imports_granularity = "Module"` is not recognized by my local rustfmt.